change/office-ui-fabric-react-2019-08-12-10-13-48-v-mare-ComboBox-fluent-updates.json

```
@@ -0,0 +1,8 @@
+{
+  "comment": "Combobox: Updated web fluent styles",
```

Fluent is by default now so we want to avoid this wording in the description for any changes in Fabric 7.0
```suggestion
  "comment": "ComboBox: updates styles to the latest design specs.",
```